lang/python/micropython-lib/patches/001-build-unix-ffi.patch

```diff
@@ -1,37 +0,0 @@
---- a/tools/build.py
```

Comment:
This patch is still necessary, otherwise the unix-ffi packages are [not included in the build](https://github.com/micropython/micropython-lib/blob/68e3e07bc7ab63931cead3854b2a114e9a084248/tools/build.py#L316-L318).

Reply:
re-included, along with your requested modifications.  Thanks!